lib/main.js

```diff
@@ -121,6 +121,12 @@ function initSession(shiny, options, shinyServer) {
 
       factory(url, ctx, (err, conn) => {
         pc.resolve(err, conn);
+        conn.onclose = e => {
```

Comment:
@jcheng5, I updated to Shiny 0.14 and deployed an app (to Connect) with `session$allowReconnect(TRUE)` enabled, but the client doesn't attempt to start a new session after the reconnect is unsuccessful. I'll try the same thing with Shiny Server.